features/gezag-minderjarige/2a.2-erkenning.feature

```diff
@@ -0,0 +1,553 @@
+#language: nl
+
+Functionaliteit: 2a.3 - erkenning
+  # Precondities voor deze vraag:
+  # | 1.1 Staat persoon (minderjarige) als ingezetene in de BRP?             | Ja                                                                                 |
+  # | 1.2 Is persoon a. minderjarig en b. niet overleden?                    | Ja                                                                                 |
+  # | 1.3 Is minderjarige naar het buitenland geëmigreerd geweest?           | Nee                                                                                |
+  # | 1.3a Is minderjarige in buitenland geboren?                            | Nee of geadopteerd met NL akte                                                     |
+  # | 1.4 Uitspraak gezag aanwezig?                                          | Nee of door een recente gebeurtenis het gezag toch (weer) van rechtswege verkregen |
+  # | 2.1 Hoeveel juridische ouders heeft de minderjarige?                   | 2                                                                                  |
+  # | 2a.1 Zijn beide ouders nu met elkaar gehuwd of geregistreerd partners? | Nee nooit                                                                          |
+  # | 2a.2 Is persoon geadopteerd met Nederlandse akte                       | Nee
+
+  # Hoe achterhalen of er erkenning is: Aktenummer Ouder1 of Ouder2 (02.81.20 of 03.81.20) bevat op de 3e positie de akteaanduiding B (erkenning bij
+  # geboorteaangifte), C (erkenning na geboorte), J (erkenning bij notariële akte) of V (gerechtelijke vaststelling ouderschap). Deze kan in de actuele categorie voorkomen maar ook 
+  # in een historische categorie (52.81.20 of 53.81.20) omdat er later nog een ander rechtsfeit geregistreerd kan zijn. De datum van erkenning kan ontleend worden aan 
+  # datum familierechtelijke betrekking (02.62.10 of 03.62.10) van dezelfde categorie als die waarin akteaanduiding B of C geconstateerd is. Kan dus ook in de
+  # historie voorkomen (52.62.10 of 53.62.10).
+
+  # Aktenummers zoals vastgelecht in de BRP en opgenomen in de lijst erkenningscodes uit de publieke tabel 39. 
+  # Zie: https://publicaties.rvig.nl/Landelijke_tabellen/Landelijke_tabellen_32_t_m_61_excl_tabel_35/Landelijke_Tabellen_32_t_m_61_in_csv_formaat/Tabel_39_Akteaanduiding
```

Comment:
Voor de toekomstbestendigheid is het beter om dit bestand te downloaden en toe te voegen aan het project, vanuit een best practice waarbij wordt aangenomen dat links niet permanent beschikbaar zullen blijven.

Reply:
overheid publicatie die buiten ons veranderd maar wijzigingen zijn relevant, ik hou het op een link en gooien we de vraag nog een keer in de groep